db/migrate/20150330002335_create_organizations.rb

```diff
@@ -2,6 +2,8 @@ class CreateOrganizations < ActiveRecord::Migration
   def change
     create_table :organizations do |t|
       t.string :name, null: false, default: ''
+      t.text :description, null: false, default: ''
+      t.string :reason_to_donate, null: false, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.